webapp/crashstats/libbugzilla.py

```diff
@@ -153,13 +151,12 @@ def crash_report_to_description(crash_report_url, processed_crash):
         frames = minidump_thread_to_frames(threads[thread_index])
 
     if not frames:
-        frames = mini_glom(
+        # Generate frames from java_exception structure
+        if frames := mini_glom(
             processed_crash,
```

Comment:
I don't think what you have would cause an error, but wouldn't `if frames :=...` replace the above `if not frames:` statement?

Reply:
I think we need the `if not frames` thing separate because we don't want to stomp on `frames` with a new value if it already has a value from `minidump_thread_to_frames`.

Reply:
Ah okay yeah I just tried this locally and it doesn't work the way I thought it would, so good to leave it as you had it.